### DISCUSSION TITLE
how to interact with a smart contract function using web3modal in react native

### DISCUSSION THREAD

**Discussion by itsnyx** [WalletConnect/walletconnect-monorepo#2414] [Developer Support] [answered]
how to interact with a smart contract function such as payable functions and normal callable functions using web3modal (wallet connect v2) in react native ?

can someone give an example of this thanks

**Comment by finessevanes** [comment, marked as answer]
Check out our examples [here](https://docs.walletconnect.com/2.0/reactnative/overview#examples).
